### PR TITLE
Updating repo link for graphql-spring-boot

### PR DESCRIPTION
**Pull request by setchy**
https://github.com/graphql-java-kickstart/graphql-spring-boot

GraphQL and GraphiQL Spring Framework Boot Starters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,61 +8,70 @@ If you want to contribute to this list (please do), send me a pull request.
 
 <!-- MarkdownTOC depth=4 -->
 
-- [Specification](#spec)
```
Comment (chentsulin) [inline README.md:11]: @setchy Could you undo these changes on TOC?

Comment (setchy) [reply README.md]: Fixed, thanks @chentsulin

**Review by chentsulin — APPROVED** [review]
